changelog.txt

```diff
@@ -1258,7 +1258,7 @@
   * more BATS tests
   * gating: clean /go/bin to install fresh tools
   * make lint: enable gocritic
-  * linter: blacklist linters instead of whitelisting
+  * linter: denylist linters instead of allowlisting
```

Comment:
nit: Should be just allowlist

Reply:
should we fix the changelog given it is really in the history? if so, im good with with @rhatdan suggestion